include/ne2000.h

```diff
@@ -244,4 +244,6 @@ class bx_ne2k_c  {
 
 };
 
+void NE2K_Init(Section* sec);
+
```

Comment:
:+1:

Reply:
Should I leave this out since it's out of PR scope? That was just there to try and fix a CI error.

Reply:
Leave your existing work as-is.
(feel free to resolve this, as there is no action or further discussion needed).